pages/business-import-export.js

```diff
@@ -25,6 +25,7 @@ const ImportExportActivities = (props) => (
       <ProcessedErrorSummary
         validatorErrors={props.validatorErrors}
         onHandleErrorClick={OnHandleErrorClick}
+        id="import_export_activities_directly_import"
```

Comment:
Passing these id's isn't actually necessary, and the onHandleErrorClick already scrolls the relevant field into view.  Just need to update the Error summary to have a dummy href so screen readers detect the link.  I'll make this change.